lib/commands/botCommands.js

```diff
@@ -37,8 +38,10 @@ function commandHandler(message) {
 
             if (containsCommand(message, commandPrompt, 'ping')) {
                 pingCommand(message);
+            } else if (containsCommand(message, commandPrompt, 'help-admin')) {
```

Comment:
Should we pass an argument to help, ie. `!help` or `!help admin` instead of a seperate `!help-admin` command?

Reply:
Hm, good point, ill leave it in for now, and add it as an issue to fix later